### ISSUE TITLE
exit code of pktctl is always zero

### ISSUE THREAD

**Issue by cjdelisle** [pkt-cash/pktd#11]
Ideal would be:
* catchall error 1
* If pktwallet is having a problem (like out of disk, corruption, etc) then it should be 128

Jeff to provide more info

**Comment by MakeItGreatAgain** [comment]
Propose adoption of BSD sysexits levels (even on non-BSD) where they do not conflict.

POSIX.1 specifies 0 for success, >1 for failure and >125 reserved, and 255 a special case that can be interpreted as 0.

Android and BSD including (iOS/OSX/FreeBSD/NetBSD/OpenBSD/Dragonfly) reserve levels >63 for specific cases - these should be adopted where they do not conflict with native reserved levels.

For example:

EX_USAGE (64)	   The command was used	incorrectly, e.g., with	the  wrong number	of arguments, a	bad flag, a bad	syntax  in a	parameter, or whatever.

     EX_DATAERR	(65)	   The input data was incorrect	in some	way.  This  should only be used for user's data and not system  files.

     EX_NOINPUT	(66)	   An input file (not a	system file) did not exist or was not readable.  This could also include errors  like	"No message" to	a mailer (if it	cared to catch it).

     EX_NOUSER (67)	   The user specified did not exist.  This might be  used	for mail addresses or remote logins.

     EX_NOHOST (68)	   The host specified did not exist.  This is used in  mail	addresses or network requests.

     EX_UNAVAILABLE (69)   A service is	unavailable.  This can occur if	a sup-  port	program	or file	does not exist.	 This can also  be used as a	catchall message when something	you   wanted to do	does not work, but you do not know   why.

     EX_SOFTWARE (70)	   An internal software	error has been detected.  This   should be limited to	non-operating system related   errors as possible.

     EX_OSERR (71)	   An operating	system error has been detected.	 This   is intended to be used for such things as "cannot   fork", "cannot create pipe",	or the like.  It in-   cludes things like getuid returning a user that   does	not exist in the passwd	file.

     EX_OSFILE (72)	   Some	system file (e.g., /etc/passwd,	   /var/run/utx.active,	etc.) does not exist, cannot	   be opened, or has some sort of error	(e.g., syntax   error).

     EX_CANTCREAT (73)	   A (user specified) output file cannot be created.

     EX_IOERR (74)	   An error occurred while doing I/O on	some file.

     EX_TEMPFAIL (75)	   Temporary failure, indicating something that	is not   really an error.  In	sendmail, this means that a   mailer (e.g.) could not create a connection,	and   the request should be reattempted later.

     EX_PROTOCOL (76)	   The remote system returned something	that was "not	   possible" during a protocol exchange.

     EX_NOPERM (77)	   You did not have sufficient permission to perform	   the operation.  This	is not intended	for file sys-	  tem problems, which should use EX_NOINPUT or   EX_CANTCREAT, but rather for	higher level permis-   sions.

     EX_CONFIG (78)	   Something was found in an unconfigured or miscon-   figured state.


In short 0 for success, >1 for failure, where levels 64-78 can be utilized as appropriate.

**Comment by MakeItGreatAgain** [comment]
Well, >0 for error, not necessarily >1 as 1 should be the default error if not more specific error fits.

Also, I did not exhaustively verify it, so it may be the case already, but ensure that all success/non-error output goes to stdout. When an exit condition >0 has been raised, all relevant output should be directed to stderr when logging to console.

**Comment by johnsonjh** [comment]
Fixed in develop https://github.com/pkt-cash/pktd/commit/6cf1da292bb45b79b592187d2784ea27bb5bb2de - OK to close.

**Comment by cjdelisle** [comment]
Thanks for noticing
